common/src/main/java/org/apache/celeborn/common/network/util/NettyUtils.java

```diff
@@ -122,14 +124,15 @@ private static PooledByteBufAllocator getSharedPooledByteBufAllocator(
         // each core should have one arena to allocate memory
         _allocator = createPooledByteBufAllocator(true, true, conf.networkAllocatorArenas());
         new NettyMemoryMetrics(
-            _allocator, "shared-pool", conf.networkAllocatorVerboseMetric(), source);
+            _allocator, "shared-pool", conf.networkAllocatorVerboseMetric(), source, null);
```

Comment:
`null` is evil, it is highly discouraged. If possible, pass an empty collection instead.

Reply:
thx, updated